helm/flowforge/templates/file-storage.yml

```
@@ -78,9 +78,7 @@ spec:
         {{ end -}}
         ports:
         - containerPort: 3001
-        securityContext:
-          allowPrivilegeEscalation: false
-          readOnlyRootFilesystem: true
+        securityContext: {{- toYaml .Values.forge.broker.containerSecurityContext | nindent 10 }}
```

This is using the broker settings

```suggestion
        securityContext: {{- toYaml .Values.forge.fileStore.containerSecurityContext | nindent 10 }}
```